docs/HowTo/Configure/Configure-HA/High-Availability.md

```diff
@@ -1,104 +1,119 @@
-description: Hyperledger Besu high availability 
+description: Hyperledger Besu high availability
 <!--- END of page meta data -->
 
-# High Availability of JSON-RPC and RPC Pub/Sub APIs
+# High availability of JSON-RPC and RPC Pub/Sub APIs
 
-To enable high availability to the [RPC Pub/Sub API over WebSockets](../../Interact/APIs/RPC-PubSub.md) 
-or the [JSON-RPC API](../../Interact/APIs/Using-JSON-RPC-API.md) run and synchronize multiple Hyperledger Besu 
-nodes to the network. Use a load balancer to distribute requests across nodes in the cluster that 
-are ready to receive requests. 
+To enable high availability to the
+[RPC Pub/Sub API over WebSockets](../../Interact/APIs/RPC-PubSub.md) or the
+[JSON-RPC API](../../Interact/APIs/Using-JSON-RPC-API.md), run and synchronize multiple Hyperledger
+Besu nodes to the network. Use a load balancer to distribute requests across nodes in the cluster
+that are ready to receive requests.
 
 ![Load Balancer](../../../images/LoadBalancer.png)
 
-!!! important 
-    We do not recommend putting [bootnodes](../../Deploy/Bootnodes.md) behind a load balancer. 
+!!! important
 
-## Determining When a Node is Ready 
+    We do not recommend putting [bootnodes](../../Deploy/Bootnodes.md) behind a load balancer.
 
-Use the [readiness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints) 
-to determine when a node is ready. 
+## Determining when a node is ready
+
+Use the
+[readiness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints) to
+determine when a node is ready.
 
 !!! note
-    The minimum number of peers and number of blocks from the best known block for a node to be considered ready
-    is deployment specific. 
 
-## Transaction Nonces 
+    The minimum number of peers and number of blocks from the best known block for determining if a
+    node considered ready is deployment specific.
+
+## Transaction nonces
 
-The nonce for the next transaction is obtained using [`eth_getTransactionCount`](../../../Reference/API-Methods.md#eth_gettransactioncount). 
-The nonce depends on the transactions in the [transaction pool](../../../Concepts/Transactions/Transaction-Pool.md).
-If [`eth_getTransactionCount`](../../../Reference/API-Methods.md#eth_gettransactioncount) and 
-[`eth_sendRawTransaction`](../../../Reference/API-Methods.md#eth_sendrawtransaction) requests for a specific account 
-are sent to multiple nodes, the [`eth_getTransactionCount`](../../../Reference/API-Methods.md#eth_gettransactioncount)
-results can be incorrect. 
+Retrieve the nonce for the next transaction using
+[`eth_getTransactionCount`](../../../Reference/API-Methods.md#eth_gettransactioncount). The nonce
```

Comment:
I think the passive voice in this paragraph (ie, how it was written originally) is needed because they are things that might be done by Besu rather than by the user (eg, if you're using EthSigner, `eth_getTransactionCount` and `eth_sendRawTransaction` aren't directly called by the user. 

Let me know if this doesn't make sense.

Reply:
Updated such that Besu obtains the nonce, as follows:
"Besu obtains the nonce for the next transaction using..."